apps/rest_api/lib/controllers/admin/user.ex

```diff
@@ -26,4 +26,20 @@ defmodule RestAPI.Controller.Admin.User do
         error
     end
   end
+
+  def show(conn, %{"id" => username} = params) do
+    params
+    |> Map.put(:username, username)
+    |> ResourceManager.get_identity()
```

Comment:
We need to refactor this get_identity.
You should pass username `nil` in order for it to work.

Reply:
@lcpojr you say to put username: nil in the above Map.put?